packages/nimble-components/src/table/template.ts

```diff
@@ -39,97 +39,112 @@ export const template = html<Table>`
             --ni-private-table-scroll-height: ${x => x.virtualizer.allRowsHeight}px;
             --ni-private-table-row-container-top: ${x => x.virtualizer.rowContainerYOffset}px;
             --ni-private-table-row-grid-columns: ${x => x.rowGridColumns ?? ''};
+            --ni-private-table-cursor-override: ${x => (x.layoutManager.isColumnBeingSized ? 'col-resize' : 'default')};
+            --ni-private-table-row-min-width: ${x => `max(100%, ${x.tableScrollableMinWidth}px)`};
+            --ni-private-table-header-row-min-width: ${x => `max(100%, calc(${x.tableScrollableMinWidth}px + ${x.virtualizer.headerContainerMarginRight}px))`};
```

Comment:
Can we move more of the css logic into the stylesheet and keep the property definition to a minimum here?
For example:
```
--ni-private-table-scrollable-min-width: ${x => `${x.tableScrollableMinWidth}px)`}
```

And then we already have headerContianerMarginRight as a property `--ni-private-table-header-scrollbar-spacer-width`

style:
```
min-width: max(100%, calc(var(--ni-private-table-scrollable-min-width) + var(--ni-private-table-header-scrollbar-spacer-width));
```

I think that also illustrates as issue with the mental model here. These css custom property definitions are trying to define bindings to property values that can be used in styles, not styles themselves. They are just making observable properties usable in CSS.

So we may want to rename `--ni-private-table-header-scrollbar-spacer-width` to something that better aligns to the property it corresponds to, ie `--ni-private-table-header-container-margin-right`.
Then it'll be easier to say in the CSS that the `.header-row` `min-wdith` is a function of the right margin and scrollable min width, etc.

Wouldn't mind a pass in this PR or a separate one that changed the names of most of these to better align to the property they are exposing. I realize it doesn't work perfectly, for example for booleans and the row grid columns, but I think we can improve the correlation a bit.